programs/marginfi/src/state/marginfi_group.rs

```diff
@@ -487,10 +490,19 @@ pub struct Bank {
     pub emissions_remaining: WrappedI80F48,
     pub emissions_mint: Pubkey,
 
+    /// For banks where `config.asset_tag == ASSET_TAG_STAKED`, this defines the last-cached
+    /// exchange rate of LST to SOL, i.e. the price appreciation of the LST. For example, if this is
+    /// 1, then the LST trades 1:1 for SOL. If this is 1.1, then 1 LST can be exchange for 1.1 SOL.
+    ///
+    /// Currently, this cannot be less than 1 (but this may change if slashing is implemented)
+    ///
+    /// For banks where `config.asset_tag != ASSET_TAG_STAKED` this field does nothing and may be 0,
+    /// 1, or any other value.
+    pub sol_appreciation_rate: WrappedI80F48,
```

Comment:
placing it here will corrupt existing banks since it will be deserialized before `collected_program_fees_outstanding`

Reply:
Also this is unnecessary now with the additional oracle adapter

Reply:
We're good for serialization as long as program fees aren't live on mainnet, but you're right we don't need this any more, will remove.